src/Infrastructure/Adapter/Adapter.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
There's only one adapter right now but this allows us to add others down the road without loss of functionality.